docs/pr_checklist.md

```diff
@@ -80,6 +80,7 @@ https://github.com/qmk/qmk_firmware/pulls?q=is%3Apr+is%3Aclosed+label%3Akeyboard
       - Encoder Configuration
       - Bootmagic Configuration
       - LED Indicator Configuration
+    - run `qmk format-json` on this file before submitting your PR
```

Comment:
I think a comment like this would be helpful to have.
```suggestion
    - run `qmk format-json` on this file before submitting your PR. Be sure to append the `-i` flag to directly modify the file, or paste the outputted code into the file. 
```

Reply:
applied, thanks